### ISSUE TITLE
Build error on OSX 10.11 - issues with ensdf fortran routines

### ISSUE THREAD

**Issue by sellitforcache** [pyne/pyne#819]
I'm trying to build the latest pyne on OSX 10.11 and I'm getting errors related intrinsics and dataype mismatches in the ensdf fortran routines...  this is the first error:

```
[  0%] Built target ReplicatePythonSourceTree
Scanning dependencies of target alphad
[  1%] Building Fortran object src/CMakeFiles/alphad.dir/ensdf_processing/ALPHAD/alphad.f.o
/Users/rmb/code/pyne-sellitforcache/src/ensdf_processing/ALPHAD/alphad.f:934:38:

                   sigdig = INT(ALOG10(hf(l)))
                                      1
Error: Type of argument 'x' in call to 'alog10' at (1) should be REAL(8), not REAL(4)
/Users/rmb/code/pyne-sellitforcache/src/ensdf_processing/ALPHAD/alphad.f:934:31:

                   sigdig = INT(ALOG10(hf(l)))
                               1
Error: Function 'alog10' at (1) is INTRINSIC but is not compatible with an intrinsic
make[2]: *** [src/CMakeFiles/alphad.dir/ensdf_processing/ALPHAD/alphad.f.o] Error 1
make[1]: *** [src/CMakeFiles/alphad.dir/all] Error 2
make: *** [all] Error 2
```

If I change "alog10" to "log10", the error is gone, but the build runs into issues with other intrinsics.  I'm not familiar with these routines, so how would you guys like to proceed with a fix?

**Comment by sellitforcache** [comment]
@rachelslaybaugh @howland

**Comment by scopatz** [comment]
@sellitforcache, what fortran compiler are you using? The file extension doesn't have a version number (eg f77); it is just f.  My mostly uninformed guess is that the default language version on your system is not the correct version for this file.  If I am not mistaken @howland wrote this, and so he and @rachelslaybaugh should know the target lang.

**Comment by makeclean** [comment]
Hi all

Lucas Jacobson discovered this recently, there are several other things that
need fixing, but for now replacing alog10 with log10 should fix the compile
problems, this tends to be only a problem for newer compilers. I would ask
Lucas @ljacobson64 to chime in.

One of the other things that needs fixing is the use of the assign keyword,
this will be deprecated in the next GCC release, why it was even allowed to
be a language feature in Fortran I don't know. But the use looks something
like this,

```
      assign 10 to m
       goto m
      10 assign 11 to m
```

etc.

This I looked into how to fix this, but it looks very very non-trivial.

Thanks

Andy

**Comment by howland** [comment]
@scopatz Alphad's manual says it was updated to f95.  I'm assuming, after @makeclean 's comment, that changing the extension wouldn't do much?  I can submit a pr that updates all the fortran source for ensdf-processing to use the proper extensions.

http://www.nndc.bnl.gov/nndcscr/ensdf_pgm/analysis/alphad/readme-alphad.pdf

The ensdf analysis programs @rachelslaybaugh and I added vary in fortran versions: f90, f95 and I think maybe some f77?

If compiling one of them is a huge hassle, I can always use the download strategy I used for three other analysis programs.  Pretty much PyNE downloads the executable from the nndc's website upon the users first attempt to use the specific analysis function.

**Comment by scopatz** [comment]
Frankly, I think that this is going to take someone who has a deeper knowledge of fortran than I do and either with a mac or with access to a mac to figure out.  I agree with you @howland that after @makeclean's comments, I don't think it makes much sense to change the extensions.Hopefully  @ljacobson64  will chime in or someone else with a mac.

> If compiling one of them is a huge hassle, I can always use the download strategy I used for three other analysis programs. Pretty much PyNE downloads the executable from the nndc's website upon the users first attempt to use the specific analysis function.

I am not sure that I fully understood this.  Was the last sentence a suggestion or the way things work now.  Sorry about my confusion :(

**Comment by ljacobson64** [comment]
I was originally trying to install PyNE and all its dependencies, including GCC, from source. I was able to get it to work just fine if I used the default version of GCC present on the machine, but when using my own build of GCC, I got errors very similar to the ones mentioned by @sellitforcache.

In regards to @makeclean 's comments, changing `ALOG10` to `LOG10` and `ALOG` to `LOG` fixes some of the problems, but not all of them. Also I did not notice any difference in the errors between GCC versions, so I don't think it's strictly a compiler version problem.

I have a branch that fixes the problems; I will PR shortly...
